compiler/backend_inkwell/src/lib.rs

```diff
@@ -60,6 +64,8 @@ pub struct CodeGen<'ctx> {
     module: Module<'ctx>,
     builder: Builder<'ctx>,
     mir: Arc<Mir>,
+    candy_value_pointer_type: PointerType<'ctx>,
+    builtins: FxHashMap<BuiltinFunction, FunctionValue<'ctx>>,
     globals: HashMap<Id, GlobalValue<'ctx>>,
     locals: HashMap<Id, BasicValueEnum<'ctx>>,
     functions: HashMap<Id, FunctionInfo<'ctx>>,
```

Comment:
```suggestion
    globals: FxHashMap<Id, GlobalValue<'ctx>>,
    locals: FxHashMap<Id, BasicValueEnum<'ctx>>,
    functions: FxHashMap<Id, FunctionInfo<'ctx>>,
```

Please also change the type of `unrepresented_ids` below. GitHub refuses to let me add a comment to it.